Dockerfile

```diff
@@ -58,3 +58,10 @@ RUN SWIFT_URL=https://swift.org/builds/$SWIFT_BRANCH/$(echo "$SWIFT_PLATFORM" |
 
 # Print Installed Swift Version
 RUN swift --version
+
+WORKDIR /code
+ONBUILD COPY Package.swift /code/
+ONBUILD RUN swift build -c release
+ONBUILD COPY Sources /code/
+ONBUILD RUN swift build -c release
+CMD ["/bin/bash", "-c", "find .build -executable -type f -not -name '*.so' | head | bash"]
```

Comment:
Done. Is it possible to use the unix file command to get the description instead?